projects/canopy/src/lib/docs/foundation/typography/typography.mdx

```diff
@@ -32,7 +32,7 @@ We use Roboto Light for headings, and Roboto Medium for emphasis.
 
 ## Expressive text
 
-The basis for the expressive scale is Lyon Display, which is our brand typeface. It should only be used in display settings at larger sizes where a greater degree of brand identity needs to be applied. You can find out more about Lyon Display, and where to get it on the <a href="https://www.legalandgeneralbrandhub.com/b/www/c/en-GB/BrandKnowledge/Guideline/2?Page=6" target="_blank">L\&G Brand Hub</a>.
+The basis for the expressive scale is Otto ABC, which is our brand typeface. It should only be used in display settings at larger sizes where a greater degree of brand identity needs to be applied. You can find out more about Otto ABC, and where to get it on the <a href="https://www.legalandgeneralbrandhub.com/b/www/c/en-GB/BrandKnowledge/Guideline/2?Page=6" target="_blank">L\&G Brand Hub</a>.
```

Comment:
I think this line will have to change. We will also need to update any reference to the L&G Brand Hub with the new link